winpr/libwinpr/clipboard/test/TestUri.c

```diff
@@ -0,0 +1,56 @@
+
+#include <stdio.h>
+#include <string.h>
+#include <memory.h>
+#include <stdlib.h>
+#include <winpr/winpr.h>
+#include "winpr/wlog.h"
+
+#define WINPR_TAG(tag) "com.winpr." tag
+#define TAG WINPR_TAG("clipboard.posix")
+
+int main()
+{
+	int nRet = 0;
+    const char* input[] = {
+		/*uri,                      file or NULL*/
+		"file://root/a.txt",        NULL,
+		"file:a.txt",               NULL,
+		"file:///c:/windows/a.txt", "c:/windows/a.txt",
+		"file:c:/windows/a.txt",    "c:/windows/a.txt",
+		"file:c|/windows/a.txt",    "c:/windows/a.txt",
+		"file:///root/a.txt",       "/root/a.txt",
+		"file:/root/a.txt",         "/root/a.txt" };
+	
+    int nLen = sizeof(input) / sizeof(char*);
+	printf("input length:%d\n", nLen / 2);
+    
+    WLog_SetLogLevel(WLog_Get(TAG), WLOG_ERROR);
+
+	for (int i = 0; i < nLen; i += 2)
+	{
+		int bTest = 0;
+		char* name = parse_uri_to_local_file(input[i], strlen(input[i]));
```

Comment:
missing declaration (previous comment was just about the `C++` guards and the `WINPR_API` export macro)